gravitee-kubernetes-client/src/main/java/io/gravitee/kubernetes/client/impl/KubernetesClientV1Impl.java

```diff
@@ -75,24 +79,67 @@ public KubernetesClientV1Impl(KubernetesConfig kubeConfig) {
         this.config = kubeConfig;
     }
 
+    @Override
+    public Maybe<Watchable> create(Watchable item) {
+        try {
+            String uri = "";
+            if (item instanceof ConfigMap configMap) {
+                uri = String.format("/api/v1/namespaces/%s/configmaps", configMap.getMetadata().getNamespace());
+            } else if (item instanceof Secret secret) {
+                uri = String.format("/api/v1/namespaces/%s/secrets", secret.getMetadata().getNamespace());
+            }
+
+            LOGGER.debug("Create resource with uri [{}]", uri);
+
+            RequestOptions requestOptions = getHTTPRequestOptions(HttpMethod.POST, uri);
+
+            ObjectMapper objectMapper = new ObjectMapper();
+            byte[] body = objectMapper.writeValueAsBytes(item);
+            return httpClient()
+                .rxRequest(requestOptions)
+                .flatMap(httpClientRequest -> httpClientRequest.rxSend(io.vertx.rxjava3.core.buffer.Buffer.buffer(body)))
+                .toMaybe()
+                .flatMap(response -> {
+                    if (response.statusCode() > 299) {
```

Comment:
3xx are not errors. Are you sure the client does not follow redirect ?

Reply:
There might be 3 responses. I can change this IF the way you prefer :-)
200 OK
201 Created
202  Accepted

Reply:
I think I'd be clearer, sorry :-D

Reply:
Hope it is fine now :-)